gym_sokoban/envs/sokoban_env.py

```diff
@@ -51,9 +51,9 @@ def seed(self, seed=None):
         self.np_random, seed = seeding.np_random(seed)
         return [seed]
 
-    def step(self, action, observation_mode='rgb_array'):
+    def step(self, action, observation_mode='raw'):
```

Comment:
Please leave the default mode at `rgb_array`. Dito for the below function calls.

Reply:
I see, reverted. However - a note for future - passing the observation mode as a parameter makes it not trivial to use through environment wrappers (i.e., in parallel). Setting the default mode in constructor would be maybe more convenient.